content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -1005,17 +1005,61 @@ commands.spawn(NodeBundle {
 
 TODO.
 
-## Animation Interpolation
+## Animation Interpolation Methods
 
-<div class="release-feature-authors">authors: @TODO</div>
+<div class="release-feature-authors">authors: @mockersf</div>
 
-TODO.
+Generally, animations are defined by their **keyframes**: snapshots of the position (and other state) or objects at moments along a time line?
```

Comment:
```suggestion
Generally, animations are defined by their **keyframes** which are snapshots of the position, other states, or objects at moments along a timeline.
```
Not sure if this change also changes the technical meaning trying to be conveyed. I also presumed the intent was to make a statement, not ask a question.

Reply:
```suggestion
Generally, animations are defined by their **keyframes**: snapshots of the position (and other state) or objects at moments along a time line.
```